image/templates/helm/shared/templates/02-scanner-v4-07-db-deployment.yaml

```diff
@@ -10,6 +10,10 @@ metadata:
     {{- include "srox.labels" (list . "deployment" "scanner-v4-db") | nindent 4 }}
   annotations:
     {{- include "srox.annotations" (list . "deployment" "scanner-v4-db") | nindent 4 }}
+    {{ - if eq ._rox.env.openshift 4 }}
```

Comment:
```suggestion
    {{- if eq ._rox.env.openshift 4 }}
```

Reply:
Oh, sharp eye! That also happened in the end statement of that if